docs/api.md

```diff
@@ -482,12 +483,14 @@ class MyAsyncRemoteTransformer extends Kinto.transformers.RemoteTransformer {
   }
 }
 
-coll.use(new MyAsyncRemoteTransformer());
+coll = kinto.collection("articles", {
+  remoteTransformers: [ new MyAsyncRemoteTransformer() ]
+});
 ```
 
 ### Multiple transformers
 
-Transformers are stacked when `#use()` is called multiple times, in the order of the calls; that means you can chain multiple encoding operations, with the decoding ones being processed in the reverse order:
+The remoteTransformers field of the plugins object passed to `Kinto#collection` is an Array. That means you can chain multiple encoding operations, with the decoding ones being processed in the reverse order:
```

Comment:
The reason I was hesitant about 'options' was that the [collection constructor](https://github.com/Kinto/kinto.js/blob/master/src/collection.js#L59) already [receives the Kinto options](https://github.com/Kinto/kinto.js/blob/master/src/index.js#L117-L121) in a variable named options. If that means that we should merge with that, I do not know. :) I do agree that there are a lot of names working together:
* options
  * adapters
  * other things...
* plugins
  * IdSchemas
  * transformers
    * remote
    * local